### DISCUSSION TITLE
Can I force Transmission to reset/re-add specific torrents?

### DISCUSSION THREAD

**Discussion by ghost** [transmission/transmission#2646] [Q&A] [answered]
The torrents which haven't completed and hit the "No data found!" error, due to the drive not being mounted for instance, tend to remain in that state permanently — I can't start or verify, the only solution appears to be remove and add the torrent file again to Transmission.

>No data found! Ensure your drives are connected or use "Set Location". To re-download, remove the torrent and re-add it.

Is there an automated way to achieve this? I don't keep copies of my downloaded torrent files and in `~/.config/transmission/torrents` they are named by the hash (I'm also above 10 000 torrents in total).

---

3.00 (bb6b5a062e)

**Comment by ckerr** [comment, marked as answer]
There is not an automated way to do this. Fixing the dreaded "No data found!" bug is on my TODO but I don't have a timeline for it.

I have a couple of suggestions, but first I want to emphasize I *know* these are not good suggestions. This isn't how the UX should be -- these suggestions are just about getting you unstuck *now* in the short term.

1. If this is only a few torrents, you can go into the Torrent Properties dialog and manually copy the info hash from the Info tab. That will tell you which file to snag from ~/.config/transmission/torrents

2.  I haven't tried this, but you may be able to do something with transmission-remote + grep to filter on error torrents and then peel the info hash to know which .torrent file in torrents/ to use
